appointments.php

```diff
@@ -4,4 +4,78 @@
 Description: A simple appointment booking plugin for WordPress.
 Version:     1.0
 Author:      Anthony Lopez
-*/
\ No newline at end of file
+*/
+
+// Exit if accessed directly.
+if (!defined('ABSPATH')) {
+    exit;
+}
+
+register_activation_hook(__FILE__, 'appointment_plugin_install');
+
+function appointment_plugin_install() {
+    global $wpdb;
+    $appointment_table = $wpdb->prefix . 'appointments';
+    $schedule_table = $wpdb->prefix . 'schedule';
+    $appointment_schedule_table = $wpdb->prefix . 'appointment_schedule';
+
+    $charset_collate = $wpdb->get_charset_collate();
+
+    // SQL to create appointments table.
+    $sql_appointment = "CREATE TABLE $appointment_table (
```

Comment:
Saca la logica de las tablas en otro archivo y manda a llamar aqui o instanciar la nueva clase ejemplo


`        $this->dbTables = new DatabaseTables($wpdb);  `